drivers/rtc/Kconfig.rv3028

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2024 ANITRA system s.r.o.
+# SPDX-License-Identifier: Apache-2.0
+
+config RTC_RV3028
+	bool "Micro Crystal RV3028 I2C extreme low power RTC driver"
+	default y
+	depends on DT_HAS_MICROCRYSTAL_RV3028_ENABLED
+	select I2C
+	help
+	  Enable the Micro Crystal RV3028 I2C RTC driver.
+
+if RTC_RV3028
+
+if RTC_ALARM || RTC_UPDATE
+
+config RTC_RV3028_THREAD_STACK_SIZE
+	int "Stack size for the RV3028 interrupt thread"
+	default 512
+	help
+	  Size of the stack used for the thread handling interrupts and dispatching callbacks.
+
+config RTC_RV3028_THREAD_PRIO
+	int "Priority for the RV3028 interrupt thread"
+	default 0
+	help
+	  Priority level for the thread handling interrupts and dispatching callbacks.
```

Comment:
You really should be using the system workqueue for handling callbacks like this, a dedicated thread for handling a single callback is really expensive, see https://github.com/zephyrproject-rtos/zephyr/blob/3570408db556fbb53cd775ae3d2e44553e28cb31/include/zephyr/kernel.h#L3356

Reply:
It was indeed an oversight on my part :) Thanks for your feedback and for pointing me to the relevant source.